src/main/java/edu/ie3/datamodel/io/naming/DataSourceMetaInformation.java

```diff
@@ -3,16 +3,16 @@
  * Institute of Energy Systems, Energy Efficiency and Energy Economics,
  * Research group Distribution grid planning and operation
 */
-package edu.ie3.datamodel.io.csv;
+package edu.ie3.datamodel.io.naming;
 
 import java.util.Objects;
 import java.util.UUID;
 
-/** Meta information, that can be derived from a certain file name */
-public abstract class FileNameMetaInformation {
+/** Meta information, that can be derived from a data source */
```

Comment:
The former implementation was more focussed on receiving those meta information by looking at a certain file name, that does comply with the specifications here. However, for the new purpose, this description does not fully meet anymore.

```suggestion
/** Meta information, that describe a certain data source */
```